basic_portfolio/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Basic Portfolio Page</title>
+<style>
+h1{
+	background-color: lavender;
+	height: 20%;
+	text-align: right;
+}
+#headertext li {
+	display: inline;
+	list-style: none;
+}
+div {
+	height: 60%;
+}
+footer {
+	background-color: lavender;
+}
+#footertext li {
+	display: inline;
+	list-style: none;
+	width: 10%;
+	text-align: center;
+}
+</style>
+</head>
+<body>
+	<h1>
+		<img src="http://news.toyark.com/wp-content/uploads/sites/4/2015/09/Koto-Force-Awakens-Stormtroopers-001-150x150.jpg"/>
+		<ul id="headertext">
+			<li>home</li>
+			<li>experience</li>
+			<li>projects</li>
+		</ul>
+	</h1>
+	<div>
+	lorem ipsum lorem ipsum lorem ipsum loren ipsum loren ipsum lorem ipsum lorem ipsum lorem ipsum loren ipsum loren ipsum
```

Comment:
lorem ipsum is a category of text, not just the words lorem ipsum.  This works too, but it doesn't give the variability of normal text so it might not be the best indicator of how it could potentially look.  Helper sites like jsbin and jsfiddle allow you to do lorem ipsum automatically by typing Lorem[#]  where [#] is a number of words you want.  If you don't want to use one of those sites, you can use something <a href="http://www.lipsum.com/" target="_blank">like this</a> to generate fake filler text.